main.cpp

```diff
@@ -374,6 +376,27 @@ void receivePing (udp_client_server::udp_server& server)
     std::cerr << "Received start signal: " << buff << "\n";
 }
 
+//r = distance, theta = heading and phi = angular displacement relative to the target
+void sendData(udpsender::vision_message *vision_writer, struct& message) {
```

Comment:
Can you move this stuff to netThread.cpp and netThread.hpp and do this on feature/angularposition not master. Master is quite outdated right now.